projects/media-viewer/src/lib/store/effects/bookmarks.effects.spec.ts

```diff
@@ -97,4 +99,26 @@ describe('BookmarksEffects', () => {
 
     expect(expectedAction).toEqual(new DeleteBookmarkFailure('error' as any));
   }));
+
+  it('should trigger update bookmarks success',  fakeAsync(() => {
```

Comment:
it is recommended to use marbles when testing effects (observables) - check other effects spec files for examples